db/migrate/20170711235422_create_articles.rb

```diff
@@ -0,0 +1,14 @@
+ROM::SQL.migration do
+  change do
+    create_table :articles do
+      primary_key :id
```

Comment:
Suggestion: What do you think about the framework generates primary key of `BIGINT` type?
http://www.mccartie.com/2016/12/05/rails-5.1.html